Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -3119,7 +3119,7 @@ static String commandNameForSelector(SEL selector)
     if (!editorState.isContentEditable)
         return;
 
-    if (editorState.isMissingPostLayoutData)
+    if (!editorState.hasPostLayoutAndVisualData())
```

Comment:
It seems like this logic only requires up-to-date post layout data, and not visual data; should this check be `hasPostLayoutData()` instead of `hasPostLayoutAndVisualData()`?

Reply:
Heh, I went over these a few times, not surprised that I still managed to miss some.